SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -178,6 +178,39 @@ bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchC
 
 		Writer->WriteValue(TEXT("max_concurrent_workers"), LaunchConfigDescription.MaxConcurrentWorkers);
 
+		// Event tracing
+		if (SpatialGDKSettings->bEventTracingEnabled)
+		{
+			Writer->WriteObjectStart(TEXT("event_tracing_configuration"));
+			Writer->WriteValue(TEXT("enabled"), true);
+
+			if (SpatialGDKSettings->bEnableEventTracingRotatingLogs)
+			{
+				Writer->WriteObjectStart(TEXT("rotating_event_log_file_configuration"));
+				Writer->WriteValue(TEXT("max_file_size_bytes"), SpatialGDKSettings->EventTracingRotatingLogsMaxFileSizeBytes);
+				Writer->WriteValue(TEXT("max_file_count"), SpatialGDKSettings->EventTracingRotatingLogsMaxFileCount);
+				Writer->WriteObjectEnd();
+			}
+			else
+			{
+				Writer->WriteObjectStart(TEXT("single_event_log_file_configuration"));
+				Writer->WriteValue(TEXT("max_file_size_bytes"), SpatialGDKSettings->EventTracingSingleLogMaxFileSizeBytes);
+				Writer->WriteObjectEnd();
+			}
+
+			Writer->WriteObjectStart(TEXT("event_filter_configuration"));
+
+			UEventTracingSamplingSettings* SamplingSettings =
+				SpatialGDKSettings->GetEventTracingSamplingSettings(); // TODO: Is this trouble in editor?
+			Writer->WriteValue(TEXT("event_pre_filter"),
+							   SamplingSettings->RuntimeEventPreFilter.Len() ? *SamplingSettings->RuntimeEventPreFilter : TEXT("false"));
+			Writer->WriteValue(TEXT("event_post_filter"),
+							   SamplingSettings->RuntimeEventPostFilter.Len() ? SamplingSettings->RuntimeEventPostFilter : TEXT("false"));
+
+			Writer->WriteObjectEnd();
```

Comment:
minor: I like to add comments that tell you what object you are finishing when writing out markup blocks.
`Writer->WriteObjectEnd(); // event_tracing_configuration`